src/terminal/parser/OutputStateMachineEngine.cpp

```diff
@@ -1859,6 +1872,37 @@ bool OutputStateMachineEngine::_GetRepeatCount(std::basic_string_view<size_t> pa
     return success;
 }
 
+// Routine Description:
+// - Parse OscSetClipboard parameters with the format `Pc;Pd`. Currently the first parameter `Pc` is
+// ignored. The second parameter `Pd` should be a valid base64 string or character `?`.
+// Arguments:
+// - string - Osc String input.
+// - content - Content to set to clipboard.
+// - queryClipboard - Whether to get clipboard content and return it to terminal with base64 encoded.
+// Return Value:
+// - True if there was a valid base64 string or the passed parameter was `?`.
+bool OutputStateMachineEngine::_GetOscSetClipboard(const std::wstring_view string,
+                                                   std::wstring& content,
+                                                   bool& queryClipboard) const noexcept
+{
+    const size_t pos = string.find(';');

```

Comment:
I supposed @DHowett is right, and we could save ourselves a bit of work with a 
```c++
if (_pfnFlushToTerminal != nullptr)
{
    return false;
}
```

here. That _should_ cause conpty to pass this through without decoding it once first. That being said, I'm pretty sure _setting the clipboard_ isn't something that's going to be done in a tight loop or end up as any sort of hot path, so I wouldn't block this PR on account of that.